lib/ember-new.js

```diff
@@ -27,6 +27,8 @@ module.exports = function(options) {
     disableDependencyChecker: true
   };
 
+  process.env["MODULE_UNIFICATION"] = !!options.isModuleUnification;
```

Comment:
Should be `EMBER_CLI_MODULE_UNIFICATION` I think 🤔...

Reply:
The tests I added are failing when updating to `EMBER_CLI_MODULE_UNIFICATION`. I'll investigate why.

Reply:
Okay, I get it! https://github.com/ember-cli/ember-cli/pull/7869 is still on ember-cli's beta branch. Next ember-cli release, it should be good. :)